discovery-infra/test_infra/helper_classes/cluster.py

```diff
@@ -167,22 +167,23 @@ def generate_and_download_infra_env(
         ssh_key=None,
         ignition_info=None,
         proxy=None,
-    ):
+    ) -> Path:
+        if self._config.is_static_ip:
```

Comment:
strange behavior to except `static_network_config` and then override it. should be `self._config.is_static_ip and not static_network_config`

Reply:
Yeah, probably missed that. It was late 😅